src/features/marketplace/lib/getCollectionName.ts

```diff
@@ -0,0 +1,25 @@
+import { KNOWN_IDS } from "features/game/types";
+import { BumpkinItem, ITEM_IDS } from "features/game/types/bumpkin";
+import { InventoryItemName } from "features/game/types/game";
+import {
+  MarketplaceTradeableName,
+  CollectionName,
+} from "features/game/types/marketplace";
+
+export function getCollectionName(
+  itemName: MarketplaceTradeableName,
+): Exclude<CollectionName, "resources"> {
+  if ((itemName as InventoryItemName) in KNOWN_IDS) {
+    return "collectibles";
+  }
+
+  if ((itemName as BumpkinItem) in ITEM_IDS) {
+    return "wearables";
+  }
```

Comment:
I think we can do away with type casting here as it's not really causing any errors with Typescript.

```suggestion
  if (itemName in KNOWN_IDS) {
    return "collectibles";
  }

  if (itemName in ITEM_IDS) {
    return "wearables";
  }
```